Makefile

```diff
@@ -5,11 +5,25 @@ CERT_MANAGER_VERSION ?= v1.11.0
 TAG ?= "$(shell date +%Y%m%d%s)"
 TAG := $(TAG)
 
+# renovate: datasource=github-tags depName=helm/helm
+HELM_VERSION ?= v3.10.2
+CHART_VERSION = v0.5.0 # x-release-please-version
+
 # RELEASE_REGISTRY is the container registry to push
 # into.
 RELEASE_REGISTRY?=ghcr.io/keptn
 ARCH?=amd64
 
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+## Tool Binaries
+KUSTOMIZE ?= $(LOCALBIN)/kustomize
+# renovate: datasource=github-tags depName=kubernetes-sigs/kustomize
+KUSTOMIZE_VERSION?=v4.5.7
```

Comment:
please put this version at the top of the file next to the other version settings